lldb/test/API/functionalities/postmortem/minidump-new/TestMiniDumpUUID.py

```diff
@@ -327,8 +327,6 @@ def test_relative_module_name(self):
         old_cwd = os.getcwd()
         self.addTearDownHook(lambda: os.chdir(old_cwd))
         os.chdir(self.getBuildDir())
-        name = "file-with-a-name-unlikely-to-exist-in-the-current-directory.so"
```

Comment:
@labath as an FYI, this test was failing because the .so was invalid. When loading manually, I still got an image list but this specific test was getting no modules when this .so was just `a`

Reply:
Yes, but this change looks like it defeats the purpose of the test. from the looks of things, this test is deliberately checking that the placeholder module is created despite the fact that the CWD contains a (bogus) file with that name -- you could confirm that by looking at the patch which introduced the change.

That said, I'm very surprised that this change makes a difference for this test, since we aren't using the dynamic linker plugin for minidumps (yet). Can you explain that?

Reply:
I had an issue where some build artifacts didn't get fully cleaned up in my script when I switched between branches. Sorry about that Pavel, I have dropped the test change.